tests/ha/control_pod/test_control_pod_restart.py

```diff
@@ -747,161 +747,97 @@ def test_mpu_after_ctrl_pod_failover(self):
     @pytest.mark.lc
     @pytest.mark.tags("TEST-40387")
     @CTFailOn(error_handler)
```

Comment:
not require

Reply:
will remove in one for whole file once all tests are done